app/uk/gov/hmrc/minorentityidentificationfrontend/models/PageConfig.scala

```diff
@@ -18,9 +18,28 @@ package uk.gov.hmrc.minorentityidentificationfrontend.models
 
 import play.api.libs.json.{Json, OFormat}
 
-case class PageConfig(optServiceName: Option[String], deskProServiceId: String, signOutUrl: String, accessibilityUrl: String)
+case class PageConfig(optServiceName: Option[String],
+                      deskProServiceId: String,
+                      signOutUrl: String,
+                      accessibilityUrl: String,
+                      labels: Option[JourneyLabels] = None)
 
 object PageConfig {
+
+
+  def apply(optServiceName: Option[String],
+            deskProServiceId: String,
+            optWelshServiceName: Option[String],
+            signOutUrl: String,
+            accessibilityUrl: String): PageConfig = {
+
+    val optLabels: Option[JourneyLabels] = optWelshServiceName match {
```

Comment:
you can use  if you want to
val optLabels: Option[JourneyLabels] = optWelshServiceName.map(welshServiceName => JourneyLabels(welshServiceName))